roles/postgresql/tasks/main.yml

```diff
@@ -80,32 +81,40 @@
     state: started
   when: postgresql_is_local
 
+- name: PostgreSQL | ensure pgpass
+  template:
+    src: pg_pass.j2
+    dest: "~/.pgpass"
+    mode: 0600
+    owner: postgres
+  changed_when: false
+  when: postgresql_is_local
+  become: true
+  become_user: postgres
+
 - name: PostgreSQL | ensure access to postgres server
-  lineinfile:
+  blockinfile:
```

Comment:
Does this mean I have to run the postgres role in addition to my app role when I deploy a new app? It was nice before I could just run the app role and it'd fix DB access.

Reply:
@tpendragon I don't think so - the pg_hba.conf file is the over-arching postgresql configuration file, which should change rarely and globally, not for each app. And changing from managing pg_hba.conf with `blockinfile` to managing it as a full template shouldn't make any difference at all to apps.

Reply:
Doesn't this change for every new application VM we add? My understanding was it's like a postgres firewall, is that not true? That's why we did the `blockinfile`, because it's used by other playbooks and executes via `delegate_to` so it'll update postgres with the new box's IP when we provision it.

Reply:
I can live with this new reality so that we can explode postgres and get a working postgres box back without running every app's playbook, but we should document that provisioning a new box that touches postgres means adding its IP here.

Reply:
Suggestion withdrawn, we can change to a template later if at all.